app/src/regular/java/com/tari/android/wallet/ui/fragment/settings/AllSettingsFragment.kt

```diff
@@ -79,7 +76,9 @@ UI tree rebuild on configuration changes"""
     lateinit var backup: WalletBackup
 
     @Inject
-    lateinit var sharedPrefsWrapper: SharedPrefsWrapper
+    lateinit var registry: SharedPrefsWrapper
```

Comment:
I like `registry` as a name, but then we should follow the same throughout the app. I think `sharedPrefs` is more readable since it's inline with the rest of the app. `sharedPrefsWrapper` is long but `sharedPrefs` is cool imo. What do you think?

Reply:
it was a merge side effect, as I had a "registry" here at the time I was merging changes with "report bug" button and "sharedPrefsWrapper", so it wasn't a rename by intent :)

but basically I intended to show what the object means in this particular scope rather than what it IS, as what it is is something we can deduct by the type